lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -442,6 +442,41 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value rhs = convertScalarToDtype(b, loc, payloadArgs[1], dtype);
     return b.create<arith::DivFOp>(loc, lhs, rhs);
   }
+  if (auto divTensorMode = dyn_cast<AtenDivTensorModeOp>(op)) {
+    AtenDivTensorModeOp::Adaptor adaptor(operands);
+    Type dtype = converter->convertType(divTensorMode.getType())
+                     .cast<RankedTensorType>()
+                     .getElementType();
+    if (!dtype.isa<mlir::FloatType>())
+      divTensorMode.emitError("unimplemented: non-floating point dtype");
+    Value lhs = convertScalarToDtype(b, loc, payloadArgs[0], dtype);
+    Value rhs = convertScalarToDtype(b, loc, payloadArgs[1], dtype);
+    Value div = b.create<arith::DivFOp>(loc, lhs, rhs);
+
+    if (divTensorMode.rounding_mode().getType().isa<Torch::NoneType>())
+      return div;
+    else {
+      std::string roundingMode;
+      if (!matchPattern(divTensorMode.rounding_mode(),
+                        m_TorchConstantStr(roundingMode)))
+        divTensorMode.emitError("only support constant str rounding mode");
+      if (roundingMode == "trunc") {
+        // "trunc" - rounds the results of the division towards zero. Equivalent
+        // to C-style integer division.
+        Value ceil = b.create<math::CeilOp>(loc, div);
+        Value floor = b.create<math::FloorOp>(loc, div);
+        Value cstZero = b.create<arith::ConstantOp>(loc, b.getZeroAttr(dtype));
+        Value pred = b.create<arith::CmpFOp>(loc, arith::CmpFPredicate::ULT,
+                                             div, cstZero);
+        return b.create<arith::SelectOp>(loc, pred, ceil, floor);
+      } else if (roundingMode == "floor") {
+        // "floor" - rounds the results of the division down. Equivalent to
+        // floor division in Python (the // operator)
+        return b.create<math::FloorOp>(loc, div);
+      } else
+        divTensorMode.emitError("invalid rounding mode");
```

Comment:
nit: if an `if` branch has braces, all branches must have braces for consistency/readability (https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements). Also, make sure to `return nullptr`.